### PR TITLE
add containsSplit function

### PR DESCRIPTION
**Pull request by Mr-Leshiy**

### CODE REVIEW

File: include/veriblock/read_stream.hpp [hunk 1]
```diff
@@ -84,6 +84,7 @@ class ReadStream {
   }
 
   size_t position() const noexcept;
+  void position(const size_t &) noexcept;
```
Comment (Warchant) [inline include/veriblock/read_stream.hpp]: maybe `setPosition` is better name?

File: src/stateless_validation.cpp [hunk 2]
```diff
@@ -1,17 +1,128 @@
...
+          (chunkDescriptorBitLength + 8 - (chunkDescriptorBitLength % 8)) / 8;
+      uint32_t waste =
+          chunkDescriptorBytesLength * 8 - chunkDescriptorBitLength;
+
+      std::vector<uint8_t> chunkDescriptorBytes =
+          buffer.readSlice(chunkDescriptorBytesLength).reverse();
+      std::vector<bool> chunkDescriptor;
+      // Read from Slice bits
+      uint32_t chunkDescriptorLength = 0;
+      for (uint32_t i1 = 0; i1 < chunkDescriptorBytes.size(); ++i1) {
```
Comment (Warchant) [inline src/stateless_validation.cpp]: for(const auto i1 : chunkDescriptorBytes)

File: include/veriblock/read_stream.hpp (hunk 1 above)
Comment (Mr-Leshiy) [reply include/veriblock/read_stream.hpp]: fixed

File: src/stateless_validation.cpp (hunk 2 above)
Comment (Mr-Leshiy) [reply src/stateless_validation.cpp]: fixed

**Pull request merged**